operations.py

```diff
@@ -0,0 +1,165 @@
+                                        verify=self.verify_ssl)
+            if response.status_code == 200 or response.status_code == 206:
+                return response.json()
+            else:
+                logger.error(response.text)
+                raise ConnectorError({'status_code': response.status_code, 'message': response.reason})
+        except requests.exceptions.SSLError:
+            raise ConnectorError('SSL certificate validation failed')
+        except requests.exceptions.ConnectTimeout:
+            raise ConnectorError('The request timed out while trying to connect to the server')
+        except requests.exceptions.ReadTimeout:
+            raise ConnectorError('The server did not send any data in the allotted amount of time')
+        except requests.exceptions.ConnectionError:
+            raise ConnectorError('Invalid endpoint or credentials')
+        except Exception as err:
+            logger.exception(str(err))
+            raise ConnectorError(str(err))
+
+
+def get_params(params):
+    params = {k: v for k, v in params.items() if v is not None and v != ''}
+    return params
+
+
+def get_output_schema(config, params, *args, **kwargs):
+    if params.get('file_response'):
+        return ({
+            "md5": "",
+            "sha1": "",
+            "sha256": "",
+            "filename": "",
+            "content_length": "",
+            "content_type": ""
+        })
+    else:
+        return ({
+            "id": "",
+            "objects": [
+                {
+                    "created": "",
+                    "description": "",
+                    "id": "",
+                    "labels": [
+                    ],
+                    "modified": "",
+                    "name": "",
+                    "object_marking_refs": [
+                    ],
+                    "pattern": "",
+                    "type": "",
+                    "valid_from": ""
+                }
+            ],
+            "spec_version": "",
+            "type": ""
+        })
+
+
+def get_api_root_information(config, params):
+    taxii = TaxiiFeed(config)
+    params = get_params(params)
+    return taxii.make_request_taxii(params=params)
+
+
+def get_collections(config, params):
+    taxii = TaxiiFeed(config)
+    params = get_params(params)
+    if params:
+        response = taxii.make_request_taxii(endpoint='collections/' + params['collectionID'] + '/')
+    else:
+        response = taxii.make_request_taxii(endpoint='collections/')
+    if response.get('collections'):
+        return response
+    else:
+        return {'collections': [response]}
+
+
+def get_objects_by_collection_id(config, params):
+    taxii = TaxiiFeed(config)
+    params = get_params(params)
+    wanted_keys = set(['added_after'])
+    query_params = {k: params[k] for k in params.keys() & wanted_keys}
+    response = taxii.make_request_taxii(endpoint='collections/' + params['collectionID'] + '/objects/',
```

Comment:
does this work for data more than 1000? doesn't seem to do pagination correctly.
use a taxii client instead. eg:

server = Server('https://limo.anomali.com/api/v1/taxii2/taxii/', user='guest', password='guest')
api_root = server.api_roots[0]
for collection in api_root.collections:
      for bundle in as_pages(collection.get_objects, per_request=1000):